package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "build": "cross-env GENERATE_SOURCEMAP=false react-scripts build",
     "postbuild": "rm -rf build/api",
-    "start": "REACT_APP_DATA_SOURCE=$DATA npm-run-all -p start:app start:lib",
+    "start": "REACT_APP_DATA_SOURCE=$DATA NODE_OPTIONS=\"--dns-result-order=ipv4first\" npm-run-all -p start:app start:lib",
```

Comment:
is this NODE_OPTIONS a mandatory field going forward ? If there is any open PR trying to resolve the network issue, it would be worth to track

Reply:
hey @ravi-kumar-pilla it seems like a work around for now. I'm not sure whether Node18 or 20 will. make any further changes to address this or not, and at the moment there's no PR opened related to this issue. Lets talk on what you think the best way to resolve this, happy to open a new issue for this